src/test/java/org/kohsuke/github/GHContentIntegrationTest.java

```diff
@@ -58,31 +62,31 @@ public void testGetFileContent() throws Exception {
         repo = gitHub.getRepository("hub4j-test-org/GHContentIntegrationTest");
         GHContent content = repo.getFileContent("ghcontent-ro/a-file-with-content");
 
-        assertTrue(content.isFile());
-        assertEquals("thanks for reading me\n", content.getContent());
+        assertThat(content.isFile(), is(true));
+        assertThat(content.getContent(), equalTo("thanks for reading me\n"));
     }
 
     @Test
     public void testGetEmptyFileContent() throws Exception {
         GHContent content = repo.getFileContent("ghcontent-ro/an-empty-file");
 
-        assertTrue(content.isFile());
-        assertEquals("", content.getContent());
+        assertThat(content.isFile(), is(true));
+        assertThat(content.getContent(), equalTo(""));
```

Comment:
I think there's a matcher for this, roughly
```suggestion
        assertThat(content.getContent(), is(emptyString()));
```